website/pages/docs/effects/box-shadow.mdx

```diff
@@ -124,6 +124,63 @@ const defaultTheme = {
 }
 ```
 
+### Dark mode
+
+The built-in shadows default to a light-mode appropriate appearance. Leverage injected CSS variables to override and customize this for other color modes, e.g.
+
+```tsx
+const defaultTheme = {
+  // ...
+  colors: {
+    shadows: {
+      xs: 'rgba(0, 0, 0, 0.05)',
+      sm: 'rgba(0, 0, 0, 0.05)',
+      defaultPrimary: 'rgba(0, 0, 0, 0.1)',
+      defaultSecondary: 'rgba(0, 0, 0, 0.06)',
+      mdPrimary: 'rgba(0, 0, 0, 0.1)',
+      mdSecondary: 'rgba(0, 0, 0, 0.06)',
+      lgPrimary: 'rgba(0, 0, 0, 0.1)',
+      lgSecondary: 'rgba(0, 0, 0, 0.05)',
+      xlPrimary: 'rgba(0, 0, 0, 0.1)',
+      xlSecondary: 'rgba(0, 0, 0, 0.04)',
+      2xl: 'rgba(0, 0, 0, 0.25)',
+      inner: 'rgba(0, 0, 0, 0.06)',
+    },
+    
+    modes: {
+      dark: {
+        shadows: {
+          xs: 'rgba(255, 255, 255, 0.05)',
+          sm: 'rgba(255, 255, 255, 0.05)',
+          defaultPrimary: 'rgba(255, 255, 255, 0.1)',
+          defaultSecondary: 'rgba(255, 255, 255, 0.06)',
+          mdPrimary: 'rgba(255, 255, 255, 0.1)',
+          mdSecondary: 'rgba(255, 255, 255, 0.06)',
+          lgPrimary: 'rgba(255, 255, 255, 0.1)',
+          lgSecondary: 'rgba(255, 255, 255, 0.05)',
+          xlPrimary: 'rgba(255, 255, 255, 0.1)',
+          xlSecondary: 'rgba(255, 255, 255, 0.04)',
+          2xl: 'rgba(255, 255, 255, 0.25)',
+          inner: 'rgba(255, 255, 255, 0.06)',
+        },
+      }
+    }
+  },
+  shadows: {
+    xs: '0 0 0 1px var(--xstyled-colors-shadows-xs)',
```

Comment:
Using `th.color()` is the way to do it. `--xstyled-colors` are internal and should not be used directly.

Reply:
I've actually never gotten that utility to work in my project